frontend/13f-app/src/components/MainSection.css

```diff
@@ -0,0 +1,23 @@
+.main-container {
+    background: url('/mnt/c/Users/eharl/documents/repos/filing13F_searcher/frontend/13f-app/src/images/img-1.jpg') center center/cover no-repeat;
```

Comment:
This is going to break when the app is deployed. See if you can replace this with a path relative to the application itself.